lua/jdtls/dap.lua

```diff
@@ -507,7 +507,7 @@ function M.fetch_main_configs(opts, callback)
             end
             local config = {
               type = 'java';
-              name = 'Launch ' .. mainclass;
+              name = 'Launch ' .. project .. ': ' .. mainclass;
```

Comment:
```suggestion
              name = 'Launch ' .. (project or '') .. ': ' .. mainclass;
```

I'm not entirely sure, but I think `project` may be `nil` on a project that doesn't use multiple modules